components/ai_chat/resources/page/components/premium_suggestion/style.module.scss

```diff
@@ -108,6 +114,10 @@
   background: var(--leo-color-container-background);
 }
 
+.priceButtonMonthly {
+  background: var(--leo-color-private-window-10);
```

Comment:
@aguscruiz is this the token that's been used in the design? If so, I think the design should be updated

We shouldn't use the semantic `private-window` token here - what if we make it bright yellow in the future (or something like that).

Reply:
Something I've just noticed, the design doesn't use the `private-window` color, it uses `primary`. So that needs to change. Every appearance of that `private-window` token in this design should be `primary`.

Check here @Douglashdaniel https://www.figma.com/design/MNwi0uwXNhkAbPiJ86G4vm/%E2%9C%A8-Leo?node-id=1753-1622&t=iGynEFjzVG5qvaIK-1

Reply:
Good point, looks like `Themable/Primary/10` https://www.figma.com/design/Kl4ISaYCPaWcIsWl5c36WA/%E2%9C%85-Leo-%5BDONE%5D?node-id=336-36734&t=l64bMc8q7ZT7CAjT-0. 

Again was just matching the current implementation 😂 https://github.com/brave/brave-core/pull/21717/files#diff-192577d71be6e255761f84d4de3391a15423267ecbb651112d1c7a96ba58fc64R8 but lets update it!

Reply:
Thanks! Weird I don't even remember ever having that as that color. Didn't notice before 😵‍💫

Thanks for updating!